x-pack/plugins/fleet/dev_docs/data_model.md

```diff
@@ -46,25 +51,42 @@ list.
 
 The total schema for actions is represented by the `FleetServerAgentAction` type.
 
+- Cleanup model: Fleet Server considers actions expired after 30 days, and will remove them via an hourly process
+- [Source](https://github.com/elastic/fleet-server/blob/9af3b2176b42a0de34c5583b5430558c03792dd0/internal/pkg/gc/schedules.go#L29-L33)
+
 ### `.fleet-actions-results`
 
+- Cleanup model: N/A
```

Comment:
Action results should probably follow a similar expiration model to actions above. It might make sense to use a more forgiving timeout, though. Maybe action results for actions that finished executing 60 or 120 days ago could be considered expired?

Reply:
Action results are actually configured as a data stream with an ILM policy in ES: https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/core/src/main/resources/fleet-actions-results-ilm-policy.json